wiki/People/The_Team/Beatmap_Nominators/General_Information/en.md

```diff
@@ -36,14 +37,20 @@ As an extension of this ability, you can also [veto a beatmap](/wiki/People/Beat
 
```

Comment:
I can't reach the text for the beatmap veto link above to suggest an edit, but it should be `[veto a beatmap](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto)` (on line 36)